src/app/zap_cluster_list.json

```diff
@@ -201,6 +202,7 @@
         "OTA_BOOTLOAD_CLUSTER": [],
         "OTA_SOFTWARE_UPDATE_PROVIDER_CLUSTER": ["ota-provider"],
         "OTA_SOFTWARE_UPDATE_REQUESTOR_CLUSTER": ["ota-requestor"],
+        "OVEN_MODE_CLUSTER": ["oven-mode-server"],
```

Comment:
There is no oven-mode-server...

Reply:
Fixed! Got carried away and forgot to list it as mode-base-server thanks for catching that!